airflow/www/static/js/grid/utils/useFilters.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { useSearchParams } from 'react-router-dom';
+
+// Params names
+export const BASE_DATE_PARAM = 'base_date';
+export const NUM_RUNS_PARAM = 'num_runs';
+export const RUN_TYPE_PARAM = 'run_type';
+export const RUN_STATE_PARAM = 'run_state';
+export const TASK_STATE_PARAM = 'task_state';
+
+const useFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const date = new Date();
+  date.setMilliseconds(0);
+  const baseDate = searchParams.get(BASE_DATE_PARAM) || date.toISOString().replace('Z', '');
+  const numRuns = searchParams.get(NUM_RUNS_PARAM) || 25;
```

Comment:
We need to swap out 25 for whatever `config.default_dag_run_display_number` is.

We can pass it via `conf.getint('webserver', 'default_dag_run_display_number')` from `/grid` in views.py

Reply:
Just modified that aswell, let me know what you think.